pkg/virt-launcher/virtwrap/converter/network.go

```diff
@@ -58,7 +58,7 @@ func CreateDomainInterfaces(vmi *v1.VirtualMachineInstance, domain *api.Domain,
 			return nil, fmt.Errorf("failed to find network %s", iface.Name)
 		}
 
-		if iface.Binding != nil || iface.SRIOV != nil {
+		if iface.Binding != nil || iface.SRIOV != nil || iface.Slirp != nil {
```

Comment:
This change seems to fit the next commit where the code is removed.

Reply:
Slirp interface has to be filtered at this point as well otherwise a redundant interface is added to the domain spec.

Reply:
This change is done in this commit because it includes Slirp plugin sidecar injection.